samples/Calculator/cppwinrt/README.md

```diff
@@ -4,7 +4,7 @@ See [../README.md](../README.md) for details of this sample.
 
 See [../csharp/](../csharp/) for a C# version of this sample.
 
-It currently targets React Native Windows 0.73.
+It currently targets React Native Windows 0.74.
```

Comment:
Can you also update the ReadMe in the parent folder? (also maybe add that to the ReadMe upgrade instructions, I also always forgot to update that one!)